content/md/first-steps/manage-your-attributes.md

```diff
@@ -130,6 +130,9 @@ Each attribute has specific properties to add more control on attribute's values
   - *Allow decimals*
   - *Minimum value*: the smallest allowed value
   - *Maximum value*: the largest allowed value
+- **Reference entity simple link & multiple link** attribute:
+  - *Related reference entity*: the reference entity with which the product is linked
+
 
 For instance, you can add a regular expression on a text field to use it for EAN codes. As they contain 13 digits, we can add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
 
```

Comment:
I added "(EE only)" line 133, rather than a tip since we don't talk about asset collection here (there are no validation or type specific parameters on asset collection)